src/main/java/FileClass.java

```diff
@@ -0,0 +1,83 @@
+import Duke.task.Deadlines;
+import Duke.task.Events;
+import Duke.task.ToDos;
+import Duke.task.TaskType;
+import Duke.task.Task;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class FileClass {
+
+    public static File tasks;
+
+    public FileClass(String filePath){
+        tasks = new File(filePath);
+        checkFile();
+    }
+
+    public void checkFile(){
+        try{
+            if(tasks.createNewFile()){
+                System.out.println("New file created for task list");
+            }else {
+                loadFile();
+            }
+        }catch (IOException e){
+            System.out.println("An error occurred: " + e.getMessage());
+        }
+    }
+
+    public void loadFile(){
+        try{
+            Scanner s = new Scanner(tasks);
+            while (s.hasNext()){
+                String command = s.nextLine();
+                String[] words = command.split("\\|",-1);
+                switch (words[0]) {
```

Comment:
What does words[0], words[1], words[2], words[3] represents in the text file? Maybe you should create variables before your switch condition to specify it, e.g: String taskType = words[0]; and so on...